### PR TITLE
adding server certificate files for API800,1000,1200

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Adding server certificate resource support for API800,1000,1200]

### Issues Resolved
[]

### Check List
- [x] New functionality includes testing.
  - [] All tests pass for go 1.11 + gofmt checks.
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: ov/server_certificate.go [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+	Name                     string            `json:"name,omitempty"`                     //"name":"null"
+	Organization             string            `json:"organization,omitempty"`             //"organization":"HPE"
+	OrganizationalUnit       string            `json:"organizationalUnit,omitempty"`       //"organizationalUnit":"EML"
+	PublicKey                utils.Nstring     `json:"publicKey,omitempty"`                //"publicKey":"2048 bits RSA Public Key"
+	SerialNumber             utils.Nstring     `json:"serialNumber,omitempty"`             //"serialNumber":"93:47:c6:5b:4d:ec:32:6e"
+	Sha1Fingerprint          utils.Nstring     `json:"sha1Fingerprint,omitempty"`          //"sha1Fingerprint":"48:32:fc:d8:6a:3f:af:71:b1:fd:5c:21:"
+	Sha256Fingerprint        utils.Nstring     `json:"sha256Fingerprint,omitempty"`        //"sha256Fingerprint":"0a:f6:b1:ca:94:c6:3a:75:97:74:e"
+	Sha384Fingerprint  utils.Nstring `json:"sha384Fingerprint,omitempty"`  //"sha384Fingerprint":"5e:3a:21:7c:78:79:6e:c6:5a:03:89:e1:"
+	SignatureAlgorithm string        `json:"signatureAlgorithm,omitempty"` //"signatureAlgorithm":"SHA256WITHRSA"
```
Comment (AnushaY1916) [inline ov/server_certificate.go:59]: Align formatting.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,36 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+
+# [v1.3.0](Unreleased) (2020-04-15)
+### Notes
+- Added resources file for Server Certificate supporting API V800, V1000, V1200
+### Oneview Features supported
```
Comment (AnushaY1916) [inline CHANGELOG.md:8]: Add only the resource which you added support for.

File: endpoints-support.md [hunk 3]
```diff
@@ -145,6 +145,12 @@
 |<sub>/rest/scope/{id}</sub>                                                              | PUT      | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/scope</sub>                                                                   | POST     | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/scope/{id}</sub>                                                              | DELETE   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+**Server Certificates**                                                                                                                         |
```
Comment (AnushaY1916) [inline endpoints-support.md:148]: API1200 is not updated.

File: examples/server_certificate.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	)
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		800,
+		"")
+	ovVer, _ := ovc.GetAPIVersion()
```
Comment (soodpr) [inline examples/server_certificate.go:25]: remove this ovVer variable as its not used in this file

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+
+# [v1.3.0](Unreleased) (2020-04-15)
```
Comment (soodpr) [inline CHANGELOG.md:5]: date is given to the released version. No need to add the date here

File: examples/server_certificate.go [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+	}
+	fmt.Println("#................... Server Certificate by Name ...............#")
+	server_certn, err := ovc.GetServerCertificateByName(server_certificate_name)
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		fmt.Println(server_certn)
+	}
+
+	server_certi, _ := ovc.GetServerCertificateByName(server_certificate_name)
```
Comment (soodpr) [inline examples/server_certificate.go:51]: again avoid calling same calls when you can reuse the server_certn for the update

Comment (nabhajit-ray) [reply examples/server_certificate.go]: Done

File: examples/server_certificate.go (hunk 4 above)
Comment (nabhajit-ray) [reply examples/server_certificate.go]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (nabhajit-ray) [reply CHANGELOG.md]: Done

**Pull request merged**